style.css

```diff
@@ -71,5 +75,226 @@ ul {
   justify-content: flex-start;
   align-items: flex-start;
   position: relative;
-  right: 45px;
+  right: 40px;
+}
+
+.tonic2 {
+  display: flex;
+  flex-direction: column;
+  min-height: 718px;
+  max-width: 300px;
+  left: 24px;
+  top: 114px;
+  border-radius: 16px;
+  padding: 16px;
+}
+
+.tnc {
+  font-family: Poppins, sans-serif;
+  font-style: normal;
+  font-weight: bold;
+  font-size: 32px;
+  line-height: 44px;
+  color: #172b4d;
+  margin-bottom: 0.5px;
+}
+
+.can {
+  font-family: Poppins, sans-serif;
+  font-style: normal;
+  font-weight: 600;
+  font-size: 13px;
+  line-height: 16px;
+  color: #344563;
+  position: relative;
+  top: 2px;
+}
```

Comment:
- [OPTIONAL] kindly ensure to implement the below change(s) requested by the previous reviewer:

Please make sure your class names are descriptive

_Originally posted by @Shaher-11 in https://github.com/isaka-lumato/portfolio/pull/2#r650194416_